### PR TITLE
1) Code indent and fout case adjustmentfor Urban module; 2) Code indent for LULCC and some corrections for URBAN and main.

### PR DESCRIPTION
**Pull request by yuanhuas**
-adj(URBAN/*.F90):
        code indent and font case adjustment for URBAN/*.F90 files.

    -adj(LULCC/*.F90,URBAN/*.F90):
        code indent for LULCC and some fix for URBAN and CoLM.F90.

### CODE REVIEW

File: main/LULCC/MOD_Lulcc_Vars_TimeVariables.F90
Comment (bnuweinan) [inline main/LULCC/MOD_Lulcc_Vars_TimeVariables.F90:1]: Code indent: Line 435-606 @yuanhuas

File: main/URBAN/MOD_Urban_Flux.F90
Comment (bnuweinan) [inline main/URBAN/MOD_Urban_Flux.F90:1]: Please confirm whether the model need stop running if imbalance happens in each balance-check part. @yuanhuas

File: main/URBAN/MOD_Urban_Shortwave.F90
Comment (bnuweinan) [inline main/URBAN/MOD_Urban_Shortwave.F90:1]: May need a short description for this file or parts of subroutines/functions. @yuanhuas

File: main/URBAN/MOD_Urban_Thermal.F90
Comment (bnuweinan) [inline main/URBAN/MOD_Urban_Thermal.F90:1]: Please confirm whether the model need stop running when energy imbalance happens or olrg < 0. @yuanhuas

File: main/URBAN/MOD_Urban_Vars_1DFluxes.F90
Comment (bnuweinan) [inline main/URBAN/MOD_Urban_Vars_1DFluxes.F90:1]: Code indent: Line 77-78, 144-145 @yuanhuas

File: main/LULCC/MOD_Lulcc_Vars_TimeVariables.F90
Comment (yuanhuas) [reply main/LULCC/MOD_Lulcc_Vars_TimeVariables.F90:1]: done.

File: main/URBAN/MOD_Urban_Flux.F90
Comment (yuanhuas) [reply main/URBAN/MOD_Urban_Flux.F90:1]: done.

File: main/URBAN/MOD_Urban_Shortwave.F90
Comment (yuanhuas) [reply main/URBAN/MOD_Urban_Shortwave.F90:1]: may be later.

File: main/URBAN/MOD_Urban_Thermal.F90
Comment (yuanhuas) [reply main/URBAN/MOD_Urban_Thermal.F90:1]: done.

File: main/URBAN/MOD_Urban_Vars_1DFluxes.F90
Comment (yuanhuas) [reply main/URBAN/MOD_Urban_Vars_1DFluxes.F90:1]: all done.

**Pull request merged**